### ISSUE TITLE
SteamVR 2.0 compatibility

### ISSUE THREAD

**Issue by dajevtic** [thestonefox/VRTK#1904]
### Environment

 * Source of VRTK Github
 * Version of VRTK Github master - Latest commit 527e622
 * Version of Unity3D 2018.2.9f1
 * Hardware used HTC Vive
 * SDK used SteamVR 2.0

### Steps to reproduce

Create a new project, add SteamVR 2.0 and VRTK

### Expected behavior

Autpopulate will find new SteamVR Setup

### Current behavior

First of all, please let me express how much I appreciate VRTK... GREAT WORK!

I have just downloaded SteamVR 2.0 which is released on the Unity Asset Store and wanted to request that VRTK can support this version, too...

If at all possible please accept this feature request.

Thank you

- VRTK does not recognize SteamVR

**Comment by dajevtic** [comment]
![image](https://user-images.githubusercontent.com/9357311/45896846-a80e6700-bdd5-11e8-9711-4255de4544c4.png)

**Comment by dajevtic** [comment]
Have you tried you this or are you writing out of the top of your head? Because this was the first thing I tried but no luck, so I'd appreciate your help on getting it to work...

Whether it will be in VRTK v3 or v4, I would just like to see this become a part of the development plan.

**Comment by threedesigner** [comment]
Hey bddckr,

VRTK is a great tool for VR development, and as dajevtic has stated, the latest version of SteamVR (2.0) doesn't appear to work with the latest version of VRTK, unfortuately. When I select SteamVR (Standalone:OpenVR) from the VRTK_SDK Setup script, it just says 'the vendor SDK for SteamVR (Standalone:Open VR) is not installed.

For clarification on your comment above, VRTK build 3.2.1 doesn't currently support SteamVR 2.0, correct? 

You say v4 will be supporting the new plugin. Any idea on a release date of v4? 


Thanks a lot and I look forward to hearing from you.

**Comment by bddckr** [comment]
Issues like these are better discussed in [![Slack](http://sysdia2.co.uk/badge.svg)](http://invite.vrtk.io) because you can instantly get answers instead of waiting on here. It's also way easier for troubleshooting in general. GitHub issues are only useful for proper bug reports with steps to reproduce in an example scene like the Issue Template requires.

**Comment by scforte773** [comment]
Im having the same issue, has anyone found a fix for it?

**Comment by bddckr** [comment]
@scforte773 Issues like these are better discussed in [![Slack](http://sysdia2.co.uk/badge.svg)](http://invite.vrtk.io) because you can instantly get answers instead of waiting on here. It's also way easier for troubleshooting in general. GitHub issues are only useful for proper bug reports with steps to reproduce in an example scene like the Issue Template requires.

**Comment by xatazch** [comment]
It is better to have this discussion here. that way people can read about it here instead of asking the same question over and over again. How is the best way to find this discussion in Slack now when the question have moved there?

**Comment by olee** [comment]
Same issue and please don't say just Slack again, because posting the relevant information here should be useful for others looking for this issue as well ;-)
So can you provide any information on when v4 might be available?
Also it might be worth posting this information on the FAQ page as well (though I saw it's already on the README.md)

**Comment by bddckr** [comment]
Any necessary info has been given above. Progress can be followed on the [Core repo's milestones](https://github.com/ExtendRealityLtd/VRTK.Unity.Core/milestones?direction=asc&sort=due_date&state=open). As always an issue tracker isn't for discussions so if you want to have discussions use Slack.

**Comment by nighgames** [comment]
@bddckr So issues like issues are better not tracked in an issue tracker that's availiable to everyone, and instead not tracked but discussed in a proprietary product that not everyone has, nor is set up in a way like an issue tracker to disseminate information and workarounds concerning a particular issue?

Well, this evaluation went a lot quicket than I thought.

**Comment by thestonefox** [comment]
VRTK v3 isn't compatible with SteamVR 2.x but v4 is. The supported SteamVR version 1.2.3 for VRTK v3 can be downloaded here: 

https://github.com/ValveSoftware/steamvr_unity_plugin/releases/download/1.2.3/SteamVR.Plugin.unitypackage

Provisional SteamVR V2 support for VRTK v3.3 has been provided by community support and can be found at: 

https://github.com/WildStyle69/SteamVR_Unity_Toolkit/tree/feat/steamvr2-input-system

**Comment by bddckr** [comment]
Please all stop reading more into my reply than what I replied with. I specifically told a user to go to Slack for specific troubleshooting help as their reply is just hijacking another ticket and an issue tracker isn't for troubleshooting. As you may know it's just two people working on this as the community isn't contributing a lot nowadays. One of the tiny requirements we have is to follow the issue template to prevent us having lots and lots of issues on a bug tracker for troubleshooting specific users' issues.
While I'd love to have google-able information available through this Issues page it just won't happen as the main community is not even on GitHub, and it ultimately ends up being me or the stone fox who are replying. The other big issue with using an issue tracker for community support is that there is no good way to offer visibility: if we close tickets people will moan, if we leave them open people will moan about the project having "hundreds of issues".

Feel free to open issues, feature requests and whatever, but do understand that the community is on Slack. You will want to pick their brains, not just ours. Apologies if my saved reply usage from half a year ago gives the impression that I don't want actual issue discussions on here. That's fine, but come pick the hive mind brain in Slack instead of waiting to never get the troubleshooting help here on GitHub, it just sadly isn't how most of our users communicate.

**Comment by aminebenichou** [comment]
Have same issue with daydream
